__tests__/app.test.js

```diff
@@ -164,4 +164,50 @@ describe('app()', () => {
       });
     });
   });
+  describe('POST', () => {
+    describe('/api/articles/:article_id/comments', () => {
+      it('201: should respond with status 201 and a body object containing the inserted comment inside the correct article (1)', () => {
+        const newComment = {
+          username: 'icellusedkars',
+          body: 'Really great work on this ticket!',
+        };
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(newComment)
+          .expect(201)
+          .then(({ body }) => {
+            const { comment } = body;
+            expect(comment).toHaveProperty('author', expect.any(String));
+            expect(comment).toHaveProperty('body', expect.any(String));
+            expect(comment.body).toBe(newComment.body);
+          });
+      });
+      it('400: should return a status 400 and a message of Bad request when a bad request is made', () => {
+        const newComment = {
+          username: 'icellusedkars',
+          body: 'Really great work on this ticket!',
+        };
+        return request(app)
+          .post('/api/articles/hello/comments')
+          .send(newComment)
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toBe('Bad request');
+          });
+      });
+      it('404: should return a status 404 and a message of Not found when it cannot post to a comment as the article does not exist', () => {
+        const newComment = {
+          username: 'icellusedkars',
+          body: 'Really great work on this ticket!',
+        };
+        return request(app)
+          .post('/api/articles/999/comments')
+          .send(newComment)
+          .expect(404)
+          .then(({ body }) => {
+            expect(body.msg).toBe('Article not found');
+          });
+      });
```

Comment:
This is a good start to your error handling, please also consider the following scenarios:
> A user makes a request with missing fields (e.g. without a username or without a body)
> A username does not exist in the database